src/app/helpers/getFormData.ts

```diff
@@ -0,0 +1,9 @@
+export const getFormData = (body: any) => {
```

Comment:
You can implement a generic type here instead of any as body is always an object